data/mainnet/tokens.js

```diff
@@ -5632,6 +5779,56 @@ module.exports = {
     ],
     cmcRewardTokenSymbols: ['iFARM', 'DEN'],
   },
+  jarvis_SES_2JPY: {
+    category: VAULT_CATEGORIES_IDS.INACTIVE_POLYGON,
+    inactive: true,
+    chain: CHAINS_ID.MATIC_MAINNET,
+    logoUrl: './icons/ses.png',
+    apyIconUrls: ['./icons/ses.png'],
+    apyTokenSymbols: ['SES'],
+    displayName: 'Jarvis: SES-2JPY',
+    subLabel: 'Auto-compounding',
+    tokenAddress: addresses.MATIC.V2.jarvis_SES_2JPY.Underlying,
+    decimals: '18',
+    vaultAddress: addresses.MATIC.V2.jarvis_SES_2JPY.NewVault,
+    priceFunction: {
+      type: GET_PRICE_TYPES.LP_TOKEN,
+      params: [addresses.MATIC.V2.jarvis_SES_2JPY.Underlying, 'SES', 'jarvis_2JPY_HODL'],
+    },
+    estimateApyFunctions: [
```

Comment:
Use regular JARVIS_V2 apy calculation here. This is not a hodl-vault. These are the input parameters for JARVIS_V2: `poolId, rewardPool, underlying, reduction`